includes/transformer/class-json.php

```diff
@@ -30,4 +30,15 @@ public function __construct( $item ) {
 
 		parent::__construct( $object );
 	}
+
+	/**
+	 * Returns the public secondary audience of this object
+	 *
+	 * @see https://www.w3.org/TR/activitystreams-vocabulary/#dfn-cc
+	 *
+	 * @return array The secondary audience of this object.
+	 */
+	protected function get_cc() {
```

Comment:
I don't quite understand why I had to add this here to keep the cc in the activity object. If I didn't the cc entry would just be empty. @pfefferle Do you know what's missing here? The cc is available in the `$item`, but it doesn't get carried over into the Activity object in `prepare_item_for_response()` ^

Reply:
This is one thing we should re-work. currently every implementer has to know how to add the `to` and `cc` to have a private, public, ... Activity. We should use the content-visibility flags, to set the audience for an Activity and the dispatcher and/or collection should set the correct `cc` and `to` based on that informations.